exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/MetricAdapter.java

```diff
@@ -102,14 +101,18 @@ private static String cleanMetricName(String descriptorMetricName) {
     final List<Sample> samples = new ArrayList<>(estimateNumSamples(points.size(), type));
 
     for (PointData pointData : points) {
-      List<String> labelNames = Collections.emptyList();
-      List<String> labelValues = Collections.emptyList();
-      Labels labels = pointData.getLabels();
-      if (labels.size() != 0) {
-        labelNames = new ArrayList<>(labels.size());
-        labelValues = new ArrayList<>(labels.size());
-
-        labels.forEach(new Consumer(labelNames, labelValues));
+      Attributes attributes = pointData.getAttributes();
+      final List<String> labelNames = new ArrayList<>(attributes.size());
+      final List<String> labelValues = new ArrayList<>(attributes.size());
+      if (attributes.size() != 0) {
+        attributes.forEach(
+            (key, value) -> {
+              String sanitizedLabelName = sanitizer.apply(key.getKey());
+              labelNames.add(sanitizedLabelName);
+              // TODO: We want to create an error-log if there is overlap in toString of attribute
+              // values for the same key name.
+              labelValues.add(value == null ? "" : value.toString());
```

Comment:
Don't believe we support null values anymore, think we may have at some point before

Reply:
should I remove the check or keep it for safety?